onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.h

```diff
@@ -152,7 +151,7 @@ class TensorrtExecutionProvider : public IExecutionProvider {
   other execution provider.
   */
   SubGraphCollection_t GetSupportedList(SubGraphCollection_t supported_nodes_list, int iterations, const int max_iterations,
-                                        const GraphViewer& graph, bool* early_termination) const;
+                                        const GraphViewer& graph, const PathString model_path, bool* early_termination) const;
```

Comment:
can this be passed by const ref?

Reply:
Sure. Changed to pointer

Reply:
If you don't expect it to be nullptr, it's better to pass it by const-ref.

Reply:
change to const PathString&